extensions/amp-orientation-observer/0.1/amp-orientation-observer.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ActionTrust} from '../../../src/action-constants';
+import {Services} from '../../../src/services';
+import {createCustomEvent} from '../../../src/event-helper';
+import {user} from '../../../src/log';
+
+const TAG = 'amp-orientation-observer';
+const DEVICE_REST_ORIENTATION_VALUES = {
```

Comment:
I'm curious how Closure Compiler processes this structure. I think for plain constants e.g. `const DEVICE_REST_ALPHA = 180;` it would inline the value, but for this dict I'm not sure what it does. It might be worth looking into the best way to store these values.

Reply:
Looked this up and you're right dicts aren't inlined. Switching to using stand alone constants.